### ISSUE TITLE
Accessibility fixes: Create ID

### ISSUE THREAD

**Issue by mariagould** [CDLUC3/ezid#544]
The following accessibility issues were identified in the Create ID form (Simple and Advanced modes)

- [x] Fix missing fieldsets for all radio buttons
- [x] Change focus so it goes to main content of page, not PDFs
- [x] On Advanced tab, focus is invisible after selecting profile in dropdown in step 4- Focus goes to Add/Delete icons before Creator Name field. Name field should come first. (Same with Title, Description, Subject, Contributor, Date, Alternate Identifier, all the rest…)
- [x] Tab order for Simple and Advanced tabs is different depending on which tab you are on. If starting with Simple, can’t tab to Advanced immediate after Simple. If on Advanced, Simple immediately precedes Advanced in tab order.

**Comment by sfisher** [comment]
Note (mostly to myself).  Trying to reorder the tabs as indicated in these tickets causes all kinds of problems with the current CSS since all the display is order dependent and requires the content shows directly after the tab.  Moving the content so the tab order is correct makes the content disappear completely.

Then it also broke some Javascript which took me a while to find out why (missing character in html).

I have the simple create tab with the correct tab order for the tabbed interface, but now the display is completely different and bad (tabs display as huge buttons with wonky colors).

It can be fixed but will take a while for me to fix css and other stuff (and I haven't yet fixed the tab order of the "PDF" sidebar) so I'm giving up for now until after the break since it is going to take more time than I thought.

In case I need to remember the files I was working with in Jan, these are the ones and the CSS is in the SCSS ui-library.

![Screenshot 2024-12-23 at 1 09 03 PM](https://github.com/user-attachments/assets/116aac46-717a-4d5c-9d0f-a1048d6dbc0e)

**Comment by adambuttrick** [comment]
Thanks for all your hard work on this @sfisher and apologies that it proved more challenging than proposed! We can pick up in the new year relative to tackling all the underlying complexities.

**Comment by sfisher** [comment]
no problem.  I'm sure I'll get it working, just changing one thing made other things break.

Have good holidays!
